### DISCUSSION TITLE
Can we use API Tokens of type Read-Only?

### DISCUSSION THREAD

**Discussion by samuelstickle** [ComfortablyCoding/strapi-plugin-slugify#72] [Q&A]
Is there a way to use a [Strapi API Token](https://docs.strapi.io/developer-docs/latest/setup-deployment-guides/configurations/optional/api-tokens.html) of type Read-Only with Slugify?

When I call fetch, I include these options:

```
{
  method: 'GET',
  headers: {
    Authorization: `bearer ${process.env.STRAPI_API_READER_TOKEN}`,
  },
}
```

If I create a token of type "Full access", the request will be recognized as authorized. But when I make a get request to a Slugify endpoint with a Read-Only token, the request is unauthorized. 

Is this the expected behavior?

**Comment by ComfortablyCoding** [comment]
No it is not expected behavior, this seems to be an issue with Strapi itself and not the slugify plugin (see strapi/strapi#14203). Once this is fixed if it is still an issue I will investigate this further.

**Reply by samuelstickle** [reply]
> No it is not expected behavior, this seems to be an issue with Strapi itself and not the slugify plugin (see [strapi/strapi#14203](https://github.com/strapi/strapi/issues/14203)). Once this is fixed if it is still an issue I will investigate this further.

Thanks. Yeah, it looks like it is an issue with Strapi itself.
